src/app/content/highlights/components/cardStyles.ts

```diff
@@ -97,6 +97,9 @@ const fadeInAnimation = css`
 `;
 
 export const mainCardStyles = css`
+  ${(props: CardProps) => !props.isFocused && (!props.data || !props.data.annotation)
```

Comment:
this is hiding unfocused cards that have no note? why did you have to add this wasn't that the behavior before?

Reply:
previously this was done at the component level (`return null`) for this statement
At some point, you asked to remove `defer` from `onFocusOutHighlight` and I found out that this approach was why `defer` was needed in the first place https://github.com/openstax/rex-web/pull/1110#discussion_r586500175
So hiding cards with css and removing `defer` worked for situations when clicking `DisplayNote` components but it didn't help when clicking `EditNote` components.
I've left `defer` in `onFocusOutHighlight` and this css part so we don't rerender cards if we don't have to, but I can roll this back if you want